src/sinks/clickhouse/integration_tests.rs

```diff
@@ -292,10 +276,10 @@ async fn no_retry_on_incorrect_data() {
     };
 
     let client = ClickhouseClient::new(host);
-    // the event contains a message field, but its being omitted to
-    // fail the request.
+    // The event contains a message field, but it's of type String, which will cause
+    // the request to fail.
     client
-        .create_table(&table, "host String, timestamp String")
```

Comment:
This test scenario no longer returns an error after upgrading to Clickhouse v22.